shared/js/background/background.es6.js

```diff
@@ -19,16 +19,4 @@
 const events = require('./$BROWSER-events.es6')
 const settings = require('./settings.es6')
 
-settings.ready().then(() => {
```

Comment:
This is MV2 background page entry point - it's not used in MV3, but since I dropped need for localStorage I cleaned it up.